src/algorithm/nest_cfgs.rs

```diff
@@ -445,14 +620,14 @@ pub(crate) mod test {
         let exit = cfg_builder.exit_block();
         cfg_builder.branch(&tail, 0, &exit)?;
 
-        let h = cfg_builder.finish_prelude_hugr()?;
+        let mut h = cfg_builder.finish_prelude_hugr()?;
 
         let (entry, exit) = (entry.node(), exit.node());
         let (split, merge, head, tail) = (split.node(), merge.node(), head.node(), tail.node());
-        // There's no need to use a FlatRegionView here but we do so just to check
-        // that we *can* (as we'll need to for "real" module Hugr's).
-        let v: SiblingGraph = SiblingGraph::try_new(&h, h.root()).unwrap();
-        let edge_classes = EdgeClassifier::get_edge_classes(&SimpleCfgView::new(&v));
+        // ALAN here's where we need a WholeHugrView...tho it's good to try SiblingMut at least somewhere
```

Comment:
This refers to the WholeHugrView in #548 .... however, even if that goes in, I'm not sure whether it'd be good to use it or not now, given the traversal algorithm in `transform_cfg_to_nested` has been specifically designed to work on a `SiblingMut`. So I should either use WholeHugrView; or stick with SiblingMut and drop this comment, and the other mentions.